tests/test_authenticated_reddit.py

```diff
@@ -83,6 +83,30 @@ def test_moderator_required__multi(self):
         sub = self.r.get_subreddit('{0}+{1}'.format(self.sr, 'test'))
         self.assertRaises(errors.ModeratorRequired, sub.get_mod_queue)
 
+    @betamax
+    def test_submission_hide_and_unhide_batch(self):
+        sub = self.r.get_subreddit(self.sr)
+        new = list(sub.get_new(limit=5, params={'show': 'all', 'count': 1}))
+
+        # Individuals first...
+        submission = new[0]
```

Comment:
Single item hiding and unhiding will already be tested via  the submission_test hide and unhide so it isn't necessary.